content/hardware/04.pro/boards/portenta-x8/tutorials/13.wordpress-webserver/content.md

```diff
@@ -64,39 +66,65 @@ services:
   Wordpress:
     depends_on:
       - db
-    image: Wordpress:latest
+    image: wordpress:latest
     volumes:
       - Wordpress_data:/var/www/html
     ports:
       - "8000:80"
     restart: always
     environment:
-      Wordpress_DB_HOST: db
-      Wordpress_DB_USER: Wordpress
-      Wordpress_DB_PASSWORD: Wordpress
-      Wordpress_DB_NAME: Wordpress
+      WORDPRESS_DB_HOST: db
+      WORDPRESS_DB_USER: Wordpress
+      WORDPRESS_DB_PASSWORD: Wordpress
+      WORDPRESS_DB_NAME: Wordpress
 volumes:
   Wordpress_data: {}
   db_data: {}
 ```
 
-Now let's create a directory on our X8 and put this **docker-compose.yml** file on our device.
+Now let's create a directory on our X8 and put this **docker-compose.yml** file on our device. You can download the file by clicking [here](assets/docker-compose.rar).
 
 ### Installing The Containers
 
-First, we create a directory where we want to add our **docker-compose.yml** file. Using the `mkdir` command we will create a directory named "wordpress-test". Navigate into this directory with a simple `cd` command. Either copy the docker-compose.yml file into this directory or create it directly here. To create the file, we can use `cat > docker-compose.yml`, this will create the file, so you can copy the content of the file from above and paste it. Push enter once to go to a new line and press `ctrl C` to exit the file editor. To copy the file from your computer onto the device use: `adb push <path to docker-compose.yml file> /home/fio/wordpress-test`.
+First, we create a directory where we want to add our **docker-compose.yml** file. Using the `mkdir` command we will create a directory named "wordpress-test". Navigate into this directory with a simple `cd` command. Either copy the docker-compose.yml file into this directory or create it directly here.
+
+To create the file, we can use `cat > docker-compose.yml`, this will create the file, so you can copy the content of the file from above and paste it. Push enter once to go to a new line and press `ctrl C` to exit the file editor. To copy the file from your computer onto the device use:
+
+```
+adb push <path to docker-compose.yml file> /home/fio/wordpress-test
+```
+
+Alternatively, you could place the `docker-compose.yml` file inside the `wordpress-test` directory and push the file using the following command:
+
+```
+adb push .\wordpress-test\ /home/fio
+```
+
+Both options work fine and depend on how you would like to handle the file.
```

Comment:
I have separated the paragraph a bit for better readability since it has a lot of commands that users need to follow, especially the main commands. Also added the last information fragment for an additional alternative which was verified while testing the code. How do you see this change? Please feel free to revert the change if you would like to.